packages/headless/src/integration-tests/field-suggestions.test.ts

```diff
@@ -198,7 +198,7 @@ describe('field suggestions', () => {
     });
 
     it('can update captions', async () => {
-      const rawValue = 'pdf';
```

Comment:
Unrelated fix: Seems this test started failing, most probably due to some content change in the index.